_data/cryptocurrencies.yml

```diff
@@ -502,6 +502,14 @@ websites:
     regions:
       - br
 
+  - name: NairaEx
+    url: https://nairaex.com/
+    img: nairaex.svg
+    tfa:
+      - email
```

Comment:
```suggestion
```
Does this refer to the "IP authorize via email" entry on the doc page? If so, it probably doesn't apply here.